feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-view/product-image-zoom-view.component.html

```diff
@@ -11,16 +11,43 @@
     </div>
     <!-- TODO: (CXSPA-7492) - Remove feature flag next major release. -->
     <div *cxFeature="'a11yKeyboardAccessibleZoom'" class="cx-image-container">
-      <cx-media
-        #defaultImage
-        class="cx-default-image-zoom"
-        (loaded)="onImageLoad()"
-        *ngIf="!isZoomed"
-        [container]="main"
-      >
-      </cx-media>
+      <ng-container *cxFeature="'useExtendedMediaComponentConfiguration'">
+        <cx-media
+          *ngIf="!isZoomed"
+          [elementType]="'img'"
+          #defaultImage
+          class="cx-default-image-zoom"
+          (loaded)="onImageLoad()"
+          [container]="main"
+        >
+        </cx-media>
+      </ng-container>
```

Comment:
query: this usage of cx-media doesn't contain format property. Is the feature toggle wrapper required?